modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/ShiftyTrimSilenceProcessor.kt

```diff
@@ -76,7 +77,8 @@ class ShiftyTrimSilenceProcessor(
         return if (enabled) inputAudioFormat else AudioFormat.NOT_SET
     }
 
-    override fun isActive(): Boolean = super.isActive() && enabled
+    @SuppressLint("MissingSuperCall")
```

Comment:
Could you add a comment with a reason for suppressing it?

Reply:
Good point. Added in [f7aae59](https://github.com/Automattic/pocket-casts-android/pull/2539/commits/f7aae59bb347cf521c4a903eb660566ea4711461).